src/documents/admin.py

```diff
@@ -13,12 +13,11 @@
 from django.utils.safestring import mark_safe
 from djangoql.admin import DjangoQLSearchMixin
 
-from documents.actions import (
-    add_tag_to_selected,
-    remove_correspondent_from_selected,
-    remove_tag_from_selected,
-    set_correspondent_on_selected
-)
+from documents.actions import (add_tag_to_selected,
```

Comment:
In the interests of consistency, please don't reformat imports.  If anything, imports should conform to an isort configuration of:

```bash
isort -m 3 --dont-skip __init__.py --virtual-env ${virtualenv}/.. --quiet ${file_name}
```